tests/oracle/test_oracle.py

```diff
@@ -199,18 +216,6 @@ def test_get_price_from_router(oracleProxySushiswap):
     usdcPriceInWeth = oracleProxySushiswap.getPriceFromRouter(usdcAddress, wethAddress)
     assert usdcPriceInEth == usdcPriceInWeth
 
-def test_get_router_for_lp_token(oracleProxySushiswap):
```

Comment:
Remove legacy method that is not used in all of lens (this method requires setting of factory addresses, which is no longer a concept)